Assets/MixedRealityToolkit.SDK/Experimental/TapToPlace/TapToPlace.cs

```diff
@@ -0,0 +1,343 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Physics;
+using Microsoft.MixedReality.Toolkit.Utilities.Solvers;
+using System.Runtime.CompilerServices;
+using UnityEngine;
+using UnityEngine.Events;
+
+[assembly: InternalsVisibleTo("Microsoft.MixedReality.Toolkit.SDK.Inspectors")]
+namespace Microsoft.MixedReality.Toolkit.Experimental.Utilities
+{
+    /// <summary>
+    /// Tap to place is a far interaction component used to place objects on a surface.
+    /// </summary>
+    public class TapToPlace : Solver, IMixedRealityPointerHandler
```

Comment:
Add experimental  attribute tag and add 
 Add [AddComponentMenu("Scripts/MRTK/Experimental/blah blah blah")]

Reply:
@Troy-Ferrell by the way, we talked with Catherine that since this is going into 2.4 not 2.3 it would be nice to actually move this out of experimental and get it into full feature right away. This is to help reduce our number of experimental features, which is growing.

Reply:
That sounds great